### ISSUE TITLE
My settings button has Disappeared

### ISSUE THREAD

**Issue by Zyclonn** [ClearVision/ClearVision-v6#46]
The Clear Vision theme has a compatibility issue with the Compact User Area theme. I love the theme but I wanted to take off a different theme but when I went to my pfp to use the settings menu it was not there. Assuming you can open the image you can see that my pfp is in the top left corner. Usually if I hover my curser over my pfp it will come up with the menu that you usually see near the bottom left hand corner (The status, mute, deafen, settings bar), except this time its to the right of my pfp. When I added the theme it just comes up with status instead of the settings bar. To top it all off my discord theme folder is hidden so I cat delete the theme I wanted to and I cant get back into the settings to turn the Clear Vision or the Compact User area themes off.

In other words I'm living with the theme on and cant take it off, It's like Minecraft's curse of binding

Pls help

![Uh oh](https://user-images.githubusercontent.com/84473005/118870167-3b183b80-b8b4-11eb-921f-fd9e8a97b0c1.png)
